src/pm/hydra/mpiexec/proxy.c

```diff
@@ -1,256 +0,0 @@
-/*
- * Copyright (C) by Argonne National Laboratory
- *     See COPYRIGHT in top-level directory
- */
-
-#include "hydra_server.h"
-
-static HYD_status alloc_proxy(struct HYD_proxy **proxy, int pgid, struct HYD_node *node);
-static HYD_status add_exec_to_proxy(struct HYD_exec *exec, struct HYD_proxy *proxy, int num_procs);
-
-HYD_status HYDU_create_proxy_list(int count, struct HYD_exec *exec_list,
-                                  struct HYD_node *node_list, int pgid, bool is_singleton,
-                                  int *proxy_count_p, struct HYD_proxy **proxy_list_p)
```

Comment:
Yes, does look like it was added by mistake then removed 😄.

Reply:
Good catch! I cleaned up before merge.